docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -1,58 +1,67 @@
 
 .. _ref-docker-with-datasource:
 
-Using the docker with an S3 bucket as remote datasource
-========================================================
+Using the Docker with a Cloud Bucket as Remote Datasource
+=========================================================
 
 Introduction
 ------------
 The Lightly Docker can be used with the Lightly Platform to do
 the following workloads in one single run:
 
-- stream your files directly from your AWS S3 bucket to your local machine without
+- stream your files directly from your cloud bucket to your local machine without
   needing to sync or download them
 - embed all images or video frames
 - sample a subset, e.g. using coreset
 - compute the metadata of the images
 - create a dataset in the Lightly Platform from the selected subset
 
-Support for streaming from Google Cloud Storage and Azure Blob Storage is
-planned. If you need any of these, write us so that we prioritize implementing it.
+Lightly supports the following cloud storage solutions:
+
+- `AWS Secure Storage Solution (S3) <https://aws.amazon.com/s3/>`_
+- `Google Cloud Storage (GCS) <https://cloud.google.com/storage>`_
+- `Azure Blob Storage (Azure) <https://azure.microsoft.com/services/storage/blobs/>`_
 
 
 Advantages
 ----------
 
 - You can run the whole Lightly workflow with one single command.
 - You can process videos directly without needing to extract frames.
-- Your data is streamed from your S3 bucket on the go instead of being first downloaded
-  to your local disk and then read from there. Thus you save both a lot of time and
+- Your data is streamed from your bucket on the go instead of being first downloaded
```

Comment:
instead of being first downloaded -> instead of it needing to be downloaded first